hack/validate/gometalinter.json

```diff
@@ -1,6 +1,7 @@
 {
   "Vendor": true,
-  "Deadline": "2m",
+  "Deadline": "8m",
```

Comment:
This can be overridden on the command line. Maybe we can set an environment variable for arm and keep the lower deadline? The environment variable can be used to set the command line flag in `hack/validate/gometalinter`

Really this should be able to run in a few seconds, so if it's this slow there is a problem somewhere.

Reply:
Indeed it takes under a minute on my (quite powerful) laptop, but about 6 minutes on a Scaleway C1 machine (which is in general pretty slow, as it's single 32-bit ARM CPU).

What about something like this:
```diff
diff --git a/hack/validate/gometalinter b/hack/validate/gometalinter
index 9830659d6..596370826 100755
--- a/hack/validate/gometalinter
+++ b/hack/validate/gometalinter
@@ -1,6 +1,18 @@
 #!/usr/bin/env bash
 set -e -o pipefail
 
+# Timeout is different depending on the arch. Values below are empirical.
+case $(uname -m) in
+       armv7l)
+               TIMEOUT=10m
+               ;;
+       *)
+               TIMEOUT=2m
+               ;;
+esac
+
 SCRIPTDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 
-gometalinter --config $SCRIPTDIR/gometalinter.json ./...
+gometalinter --deadline "$TIMEOUT" --config $SCRIPTDIR/gometalinter.json ./...
```

Reply:
That could work, but is the timeout really dependent on the arch, or is it more about the particular host?

Maybe we can just do `--deadline "${GOMETALINTER_TIMEOUT-2m}"` and set that env value in  `Dockerfile.armhf` ?

Reply:
OK, will redo. I also discovered that a run on AARCH64 does not fit into 2m either.